openc3-init/plugins/packages/openc3-accessor-test/targets/ACCESS/target.txt

```diff
@@ -0,0 +1,4 @@
+IGNORE_PARAMETER ID_ITEM
+
+CMD_UNIQUE_ID_MODE
+TLM_UNIQUE_ID_MODE
```

Comment:
Are these required for the new accessor modes? Or if you have the exact same ID_ITEM KEY then no?

Reply:
They are not required if all the packets have the same Accessor, ID_ITEMS, and keys